Source/WebKit/UIProcess/WebAuthentication/fido/CtapAuthenticator.cpp

```diff
@@ -239,10 +252,41 @@ void CtapAuthenticator::continueMakeCredentialAfterResponseReceived(Vector<uint8
 }
 
 void CtapAuthenticator::getAssertion()
+{
+    CTAP_RELEASE_LOG("getAssertion");
+
+    auto options = std::get<PublicKeyCredentialRequestOptions>(requestData().options);
```

Comment:
Again, most other uses of `std::get<PublicKeyCredentialRequestOptions>` use a reference (although this is inconsistent). Should we make this copy, since we don't seem to be modifying it?